test/fixtures/removal.ts

```diff
@@ -3,7 +3,7 @@ import type { UnpackedRemovalIdV0Struct } from '@/typechain-types/contracts/Remo
 export const defaultRemovalTokenIdFixture: UnpackedRemovalIdV0Struct = {
   idVersion: 0,
   methodology: 1,
-  methodologyVersion: 1,
+  methodologyVersion: 0,
```

Comment:
what's the reasoning behind starting this as 0, but not methodology? Or does this map to the current actual methodology version?

Reply:
this maps to the current actual methodology version